plugins/woocommerce/client/blocks/assets/js/blocks/cart/test/editor-integration.tsx

```diff
@@ -131,6 +132,12 @@ describe( 'Cart block editor integration', () => {
 		expect( emptyCartBlock ).toBeInTheDocument();
 		expect( emptyCartBlock ).toHaveAttribute( 'hidden' );
 
+		await waitFor( () => {
+			expect(
+				screen.getByLabelText( /Block: Filled Cart$/i )
+			).toBeInTheDocument();
```

Comment:
I generally recommend `toBeVisible()` when it's about locating an element that is visible to the user. While the difference is subtle, the test sees it, but not necessary the user, right? So if we want to be sure the user sees it, we should prefer `toBeVisible()`. 

Note that this is only for positive queries (the element is visible). When the element doesn't exist, we should still be using `.not.toBeInTheDocument()` since `toBeVisible()` will throw errors.